elixir/apps/domain/priv/repo/seeds.exs

```diff
@@ -348,18 +348,21 @@ IO.puts("")
       name: "FZ User iPhone",
       external_id: Ecto.UUID.generate(),
       public_key: :crypto.strong_rand_bytes(32) |> Base.encode64(),
-      last_seen_user_agent: "iOS/12.7 (iPhone) connlib/0.7.412"
+      last_seen_user_agent: "iOS/12.7 (iPhone) connlib/0.7.412",
+      identifier_for_vendor: "APPL-#{Ecto.UUID.generate()}"
     },
     unprivileged_subject
   )
 
 {:ok, _admin_iphone} =
   Domain.Clients.upsert_client(
     %{
-      name: "FZ Admin iPhone",
+      name: "FZ Admin Laptop",
       external_id: Ecto.UUID.generate(),
       public_key: :crypto.strong_rand_bytes(32) |> Base.encode64(),
-      last_seen_user_agent: "iOS/12.7 (iPhone) connlib/0.7.412"
+      last_seen_user_agent: "Mac OS/14.5 (MacBook) connlib/0.7.412",
+      device_serial: "AAPL-SERIAL-#{Ecto.UUID.generate()}",
+      device_uuid: "AAPL-UUID-#{Ecto.UUID.generate()}"
```

Comment:
```suggestion
      device_serial: "FVFHF246Q72Z",
      device_uuid: "#{Ecto.UUID.generate()}"
```

Minor, but these would be the realistic-ish values (to use for visual formatting / design)